app/styles/topbar.scss

```diff
@@ -112,8 +117,8 @@
       &__options {
         top: $topbar-height;
         left: 0;
-        min-width: 250px;
-        z-index: 2;
+        min-width: $topbar-dropdown-width;
+        z-index: 5;
```

Comment:
soo, setting a z-index usually means that you're handling the interaction between two different elements. A comment describing why the zindex is needed would be great.